### PR TITLE
Setup tests environment

### PR DESCRIPTION
**Pull request by yashka713**
Please, check it and prove.

### CODE REVIEW

**Review by PetroKuida — APPROVED** [review]
Everything seems fine.

File: .gitignore [hunk 1]
```diff
@@ -13,4 +13,8 @@
 .idea
 /.byebug_history
 
+<<<<<<< HEAD
+\.env
+=======
 .env
+>>>>>>> master
```
Comment (Vitaliy-Maznyi) [inline .gitignore]: will "<<<<<<" be on master after pull request is merged?

File: spec/factories/factories_user.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+FactoryGirl.define do
+  factory :user do
+    first_name 'Tony'
+    last_name 'Stark'
+    email 'tonystark@example.com'
+    phone '+380310000000'
+    gender true
+    date_of_birth DateTime.new(1990, 1, 1)
```
Comment (Vitaliy-Maznyi) [inline spec/factories/factories_user.rb]: please change to birth_date cause it won't work

File: spec/factories/factories_user.rb [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+    admin false
```
Comment (Vitaliy-Maznyi) [inline spec/factories/factories_user.rb]: please define inside 'factory :user do' :
 ```
factory :admin do
      admin true
    end
```
it makes it easy to create admin user for tests using `create :admin`

**Review by Vitaliy-Maznyi — CHANGES_REQUESTED** [review]
create please factory for appartments

File: spec/factories/factories_user.rb (hunk 2 above)
Comment (yashka713) [reply spec/factories/factories_user.rb]: thx, I really didn't see that :+1:

File: .gitignore [hunk 4]
```diff
@@ -13,4 +13,6 @@
 .idea
 /.byebug_history
 
+\.env
```
Comment (Vitaliy-Maznyi) [inline .gitignore]: why do you need this?

Comment (DenisUA) [reply .gitignore]: @yashka713 Please remove this

**Review by DenisUA — APPROVED** [review]
Looks good

**Pull request merged**